arm-software/embedded/arm-runtimes/test-support/generic-test-wrapper.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: Copyright 2023-2024 Arm Limited and/or its affiliates <open-source-office@arm.com>
+
+# This is generic test wrapper, used to execute binaries either with QEMU or FVP. 
+# Underneath implementation is transparent to the user, so user need not worry 
```

Comment:
Since the user has to provide `--test_executor` and the rest of QEMU/FVP specific arguments, this description does not seem to be correct - it appears that the purpose of this script is something different.